Examples/ObjectiveC/UserLocationAnnotationExample.m

```diff
@@ -33,7 +33,7 @@ - (void)update {
 
 - (void)updateHeading {
     // Show the heading arrow, if the heading of the user is available.
-    if (self.userLocation.heading.trueHeading) {
+    if (self.userLocation.heading.trueHeading > 0) {
```

Comment:
Can this go negative?

Reply:
Correct values for [`CLHeading.trueHeading`](https://developer.apple.com/documentation/corelocation/clheading/1423568-trueheading?language=objc) are `0` to `360`, with negative meaning there is no heading available (which is what this is checking for).